### PR TITLE
[TS] LPS-124963 Renaming to OpenIdConnectSessionProvider

### PR DESCRIPTION
**Pull request by stian-sigvartsen**
Hi Brian. Re. https://github.com/brianchandotcom/liferay-portal/pull/98816#issuecomment-780007480

Explanation of why *Provider is an appropriate name... https://github.com/brianchandotcom/liferay-portal/pull/98816#issuecomment-780945884

### CODE REVIEW

File: modules/apps/portal-security-sso/portal-security-sso-openid-connect-impl/src/main/java/com/liferay/portal/security/sso/openid/connect/internal/provider/OpenIdConnectSessionProviderImpl.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
+/**
```
Comment (liferay-appsec) [inline modules/apps/portal-security-sso/portal-security-sso-openid-connect-impl/src/main/java/com/liferay/portal/security/sso/openid/connect/internal/provider/OpenIdConnectSessionProviderImpl.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

**Review by liferay-appsec — COMMENTED** [review]
This is an automatic review suggestion

**Comment by stian-sigvartsen** [comment]
ci:forward

**Comment by stian-sigvartsen** [comment]
ci:test:relevant

**Comment by stian-sigvartsen** [comment]
ci:test:relevant

**Comment by stian-sigvartsen** [comment]
ci:forward

**Pull request closed without merging**